apps/docs-site/src/css/custom.css

```
@@ -148,6 +156,37 @@
+ *    `border-radius` on the container itself but allow popovers to escape.
+ *
+ * 2. `.playgroundPreview` is `position: static`, so any absolute-positioned
+ *    descendant (the DatePicker popover, positioned by Floating UI) finds
+ *    its containing block all the way up at the viewport. Without a
+ *    constrained containing block, `width: max-content` on the popover
+ *    resolves against the viewport and the calendar table stretches edge
+ *    to edge across the page — see issue #35.
+ *
+ *    `position: relative` here makes the preview area the popover's
+ *    containing block, so its width is bounded.
+ * =======================================================*/
```

Stylelint (17.9.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)